src/core/send.c

```diff
@@ -872,6 +872,52 @@ QuicSendWriteFrames(
         }
     }
 
+    if (Send->SendFlags & QUIC_CONN_SEND_FLAG_BIDI_STREAMS_BLOCKED) {
```

Comment:
Move these up after the `if ((Send->SendFlags & QUIC_CONN_SEND_FLAG_MAX_STREAMS_UNI)) {` condition.